src/authentication-strategies/JWT.strategy.ts

```diff
@@ -0,0 +1,35 @@
+const jwt = require('jsonwebtoken');
+import {promisify} from 'util';
+const verifyAsync = promisify(jwt.verify);
+// Consider turn it to a binding
+const SECRET = 'secretforjwt';
+import {Request, HttpErrors} from '@loopback/rest';
+import {UserProfile} from '@loopback/authentication';
+import * as _ from 'lodash';
+import {AuthenticationStrategy} from './authentication.strategy';
+
+export class JWTStrategy implements AuthenticationStrategy {
+  async authenticate(request: Request): Promise<UserProfile | undefined> {
+    let access_token = request.query.token || request.headers['authorization'];
```

Comment:
@jannyHou I meant to say we should check `request.query.access_token`. The variable name can stay as `token` or `accessToken`.

Reply:
got it.